src/lib/cli.ts

```diff
@@ -61,6 +69,16 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+const notifyNewVersion = (update: updateNotifier.UpdateInfo) => {
+    const changelogUrl: string = `https://sonarwhal.com/about/changelog.html`;
+    // No indentation due to the use of `\` to avoid new line.
+    // https://stackoverflow.com/a/35428171
+    const message: string = `Update available ${chalk.red(update.current)}${chalk.reset(' → ')}${chalk.green(update.latest)}\
+\nPlease update to the newest version ${chalk.green(update.latest)}\nSee ${chalk.cyan(changelogUrl)} for details`;
```

Comment:
Do we need both `Update available...` and `Please update to the newest version`? To me they basically specify the same thing.